x-pack/test/functional/apps/dashboard/reporting/lib/compare_pngs.ts

```diff
@@ -23,8 +20,8 @@ export async function checkIfPngsMatch(
   const sessionDirectoryPath = path.resolve(screenshotsDirectory, 'session');
   const failureDirectoryPath = path.resolve(screenshotsDirectory, 'failure');
 
-  await mkdirAsync(sessionDirectoryPath, { recursive: true });
-  await mkdirAsync(failureDirectoryPath, { recursive: true });
+  await fs.mkdir(sessionDirectoryPath, { recursive: true });
+  await fs.mkdir(failureDirectoryPath, { recursive: true });
```

Comment:
We have similar "promisifed" fs method usage all over Kibana.

I think it's great that Node 14 gives us fs promises out-of-the-box, but just out of curiosity, what made you decide to update this here and not anywhere else?

Reply:
The core promise API have been there since Node.js v10, so using that isn't directly related to this PR. However there were some TypeScript related issues with the old implementation in this file and instead of trying to figure out how to fix it, I simply used to native API instead which didn't have any TypeScript issues.